alignak/http_daemon.py

```diff
@@ -243,6 +244,10 @@ def get_socks_activity(self, socks, timeout):
     # We are asking us to stop, so we close our sockets
     def stop(self):
         self.stop_requested = True
+        for t in self.threads:
+            while t.is_alive():
+                t.join(5)
```

Comment:
and eventually if we would had a way to interrupt / stop such an http client thread that would be the best, but it's something else.. another PR ;)